src/api/services/usersService.ts

```diff
@@ -111,6 +111,17 @@ export const getAllUsersFromParticipantWithRoles = async (participant: Participa
   return mapUsersWithParticipantRoles(usersWithParticipants, participant.id);
 };
 
+export const getUserRolesForCurrentParticipant = async (participant: Participant, user: User) => {
+  const userToParticipantRoles = await UserToParticipantRole.query()
+    .where('participantId', participant.id)
+    .where('userId', user.id);
+
+  const userRoles = await UserRole.query();
+  return userToParticipantRoles.map((role) => {
```

Comment:
The return type of this method is `Promise<(UserRole | null)[]>`, partly because of the null coalescing operator on the line below. I think we can avoid both of these things by doing the following:
```
  return userToParticipantRoles
    .map((role) => userRoles.find((userRole) => userRole.id === role.userRoleId))
    .filter((role): role is UserRole => role !== undefined);
```